src/Simulation/Simulators/QCTraceSimulator/QCTraceSimulator.Primitive.Measure.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is this file new? There seems to be a change to how Measure works for the tracer. Can you elaborate a bit on that?

Reply:
Yes, previously QCTraceSimulator defined an override for `M` instead of `Measure` and relied on the behavior that any C# implementation would automatically override the Q# even if it wasn't body intrinsic (`M` has always been defined as a decomposition on `Measure` but the simulators would unconditionally override that). Now that the built-in simulators cannot inherit from and directly override the callabes from the target package, this simulator needs to actually participate in the decomposition+intrinsic infrastructure, meaning it needs to provide an implementation for `Measure` instead.